buildenv/docker/mkdocker.sh

```diff
@@ -329,6 +354,27 @@ fi
   echo "    xz \\"
   echo "    zip \\"
   echo "    zlib-devel \\" # required by git, python
+if [ $criu != no ] ; then
+  echo "    asciidoc \\"
+  echo "    libaio-devel \\"
+  echo "    libbsd-devel \\"
+  echo "    libcap-devel \\"
+  echo "    libnet-devel \\"
+  echo "    libnl3-devel \\"
+  echo "    iproute-devel \\"
+  echo "    nftables \\"
+  echo "    pkgconfig \\"
+  echo "    protobuf \\"
+  echo "    protobuf-c \\"
+  echo "    protobuf-c-devel \\"
+  echo "    protobuf-compiler \\"
+  echo "    protobuf-devel \\"
+  echo "    protobuf-python \\"
+  echo "    protobuf-static \\"
+  echo "    python-ipaddress \\"
+  echo "    python3-future \\"
+  echo "    xmlto \\"
+fi
```

Comment:
Are these needed only to compile CRIU? If so, the generated dockerfile should be restructured so these extra packages are not in the final image.